app/templates/components/header-bar.hbs

```diff
@@ -1,5 +1,5 @@
 <div class="{{if orgauthorizationService.isInTrustedOrg 'trusted-org-header'}} {{if showMe 'active'}} user-menu-right"  title="My Information" {{action "toggleMe"}} >
-  <img src={{profileImg}} class="img-circle profile-image"/>
+  <img style="background: url({{profileImg}})" class="img-circle profile-image"/>
```

Comment:
This won't fix the issue. The problem is that the relative path used for the image is wrong. Compare the path that this image is using with the logo image path on the DCA site.

Reply:
Logo is handled with `url()` in css [here](https://github.com/MyPureCloud/developer-tools/blob/master/app/styles/navbar.less#L11). The path I used was wrong though, I have fixed it in the last commit.